test/controllers/update/update.interrupt.card.controller.spec.ts

```diff
@@ -65,25 +65,20 @@ describe("UPDATE INTERRUPT CARD controller", () => {
       expect(resp.text).toContain(BEFORE_START_PAGE_LINK_VERIFICATION);
     });
 
-    test(`renders the update-interrupt-card page with back link to update-any-trusts-involved if flag off`, async () => {
-      mockIsActiveFeature.mockReturnValueOnce(false);
+    test(`renders the update-interrupt-card page with back link to update-any-trusts-involved`, async () => {
```

Comment:
the description says it should back link to update-any-trusts-involved, but test checks back link is SECURE_UPDATE_FILTER_URL

Reply:
Updated descriptions for both tests changed in this file.